commit 4664cb8b0cb9c25d37c5c86e74af757d1bd79e13 (refs/changes/18/2818/3)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-10-19T19:21:22+02:00 (2 years ago)
    
    TOOLS-1899 sdcadm doesn't build with node v4.8.4 due to lockfd
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/package.json b/package.json
index 940683d..957a050 100644
--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
     "kthxbai": "~0.4.0",
     "joyent-schemas": "git+https://github.com/joyent/schemas.git#385e6eb",
     "jsprim": "1.2.2",
-    "lockfd": "git+https://github.com/trentm/node-lockfd.git#fcbc0e2",
+    "lockfd": "1.0.1",
     "mkdirp": "0.3.5",
     "node-uuid": "1.4.1",
     "once": "1.3.0",
